.github/workflows/conda-forge-ci.yml

```diff
@@ -0,0 +1,73 @@
+name: C++ CI Workflow with conda dependencies
+
+on:  
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Execute a "nightly" build at 2 AM UTC
+  - cron:  '0 2 * * *'
+
+jobs:
+  build:
+    name: '[${{ matrix.os }}@${{ matrix.build_type }}@conda]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build_type: [Release]
+        os: [ubuntu-latest, windows-2019]
```

Comment:
Why `windows-2019` and not `windows-latest`?

Reply:
To test against Visual Studio 2019. `windows-latest` only has Visual Studio 2022, and robotology-superbuild still uses Visual Studio 2019. The vast majority of problems on Windows is using something that is not supported on older versions, either for bugs that are fixed in modern versions or due using some new features that are implemented in newer versions. For this reason, I prefer to test VS2019.

Reply:
Got it, thanks!